content/articles/looking-at-payments.md

```diff
@@ -0,0 +1,100 @@
+Category: Product
+Authors: Vincent
+
+
+As we are working on our product (if you work in an agency, we'd love to have a chat with you! send us a mail at team AT wearewizards.io), I was curious to see the payments landscape in Europe, having read so much about [#VATMESS](https://twitter.com/hashtag/vatmess).
+<!-- PELICAN_END_SUMMARY -->
+
+Most Software As A Service business operate with monthly plans with various tiers. 
+In Europe, before 2015, the supplier would charge their own VAT rate to their customers: for example if the supplier was based in the UK and the customer in France, the supplier would charge the UK VAT rate.
+
+since 2015 however, the supplier now has to charge the VAT rate where the customer is located and report sales to that country. This change was made to prevent unfair competitions from countries with a lower VAT rate, with big companies moving to Luxembourg for example. Regulators only forgot to set a lower limit (they are apparently now thinking of a £5k limit) and didn't think of SME or thought about it and decided that crippling european businesses would be a good idea.
+Note that this change only applies to B2C customers.
+
+This article will focus on what it means for a UK based VAT registered company as it is our case.
+
+
+# Business side
+The UK has put in place something called the VAT Mini One Stop Shop or VAT MOSS (https://www.gov.uk/guidance/register-and-use-the-vat-mini-one-stop-shop). This allows businesses to register in only one country and let it redistribute the money after filing a VAT return for other countries on a quarterly basis.
+
+If the customer is a business and is VAT registered, just getting the VAT number and checking it should be enough. However, companies paying without giving their VAT number or non-VAT registered ones should be treated as B2C.
+
+As mentioned in the introduction, we need to charge the VAT of the customer which means we need to locate it.
+
+Here are some examples of pieces of informations we can use:
+
+- billing address
+- ip address geolocation
+- bank location
+- card location
+
+Note that the legislation requires to have 2 non-conflicting informations and keep that for 10 (!) years. 
+If you don't manage that, you are supposed to reject the transaction. You could possibly ask for the user to send some proof by mail but that's a terrible user experience.
+
+If we manage to get customers to pay us, we then need to separate sales by country and apply the right VAT. This gives us the amount of tax we need to pay for each country.
+
+# Coding
+Now that we have an overview of what the VAT system looks like, let's have a look at the solutions.
+
+## Payment providers
+None of these actually help with the VAT mess but are still needed.
+There are a few alternatives here:
+
+- [Stripe](https://stripe.com)
+- [Braintree](https://www.braintreepayments.com/)
+- [Paymill](https://www.paymill.com/)
```

Comment:
I linked to a provider that did vat moss a while back, can you remember which one that was?